AmplifyPlugins/DataStore/AWSDataStoreCategoryPluginTests/Core/SQLStatementTests.swift

```diff
@@ -836,4 +877,17 @@ class SQLStatementTests: XCTestCase {
         XCTAssertEqual(variables[4] as? String, "%gelato%")
         XCTAssertEqual(variables[5] as? String, "ice cream%")
     }
+
+    func testTranslateQueryPredicateWithNameSpaceWhenFieldNameSpecified() {
+
+        let predicate = field("commentPostId") == "postID"
+
+        let statement = ConditionStatement(modelSchema: Post.schema, predicate: predicate, namespace: "root")
+        XCTAssertEqual("""
+          and "root"."commentPostId" = ?
+        """, statement.stringValue)
+
+        let variables = statement.variables
+        XCTAssertEqual(variables[0] as? String, "postID")
+    }
```

Comment:
With XCTAssertEqual, I believe the second parameter is suppose to be the expected value.  how about we change this code to match the other functions in this file:
```
    func testTranslateQueryPredicateWithNameSpaceWhenFieldNameSpecified() {
        let predicate = field("commentPostId") == "postID"

        let statement = ConditionStatement(modelSchema: Post.schema, predicate: predicate, namespace: "root")

        let expectStatement =
            """
              and "root"."commentPostId" = ?
            """
        let expectedVariable = "postID"
        let variables = statement.variables
        XCTAssertEqual(statement.stringValue, expectStatement)
        XCTAssertEqual(variables[0] as? String, expectedVariable)
    }
```